cmd/controller-manager/OWNERS

```diff
@@ -0,0 +1,66 @@
+approvers:
+- deads2k
+- lavalamp
+- mikedanese
+- sttts
+reviewers:
```

Comment:
This list is too wide.  Add the usual suspects from apimachinery.  This is about infrastructure and wiring, so I don't think sig-apps much cares.

Reply:
done